### PR TITLE
testcase/task_manager, wifi_manager: modify giving 1st argument of TC macro

### PR DESCRIPTION
**Pull request by jeongchanKim**
1st argument of TC macro should be function name which is tested by testcase, not testcase name

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d436e85c123551782cb82d2ebae1743b673c63de] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d436e85c123551782cb82d2ebae1743b673c63de] - Code Rule Check (C++) OK.

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 1]
```diff
@@ -188,7 +188,7 @@ static void utc_wifi_manager_get_mode_n(void)
 
 	ret = wifi_manager_get_info(info);
 
-	TC_ASSERT_EQ("wifi_manager_get_mode_n", ret, WIFI_MANAGER_INVALID_ARGS);
+	TC_ASSERT_EQ("wifi_manager_get_mode", ret, WIFI_MANAGER_INVALID_ARGS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: get_info

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 2]
```diff
@@ -201,8 +201,8 @@ static void utc_wifi_manager_get_mode_p(void)
 
 	ret = wifi_manager_get_info(&info);
 
-	TC_ASSERT_EQ("wifi_manager_get_mode_p", ret, WIFI_MANAGER_SUCCESS);
-	TC_ASSERT_EQ("wifi_manager_get_mode_p", info.mode, STA_MODE);
+	TC_ASSERT_EQ("wifi_manager_get_mode", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: get_info

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 3]
```diff
@@ -212,19 +212,19 @@ static void utc_wifi_manager_get_mode_p(void)
 					strlen(CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SOFTAP_PASSPHRASE) + 1);
 	ret = wifi_manager_set_mode(SOFTAP_MODE, &ap_config);
 
-	TC_ASSERT_EQ("wifi_manager_get_mode_p", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("wifi_manager_get_mode", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: set_mode

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 4]
```diff
@@ -212,19 +212,19 @@ static void utc_wifi_manager_get_mode_p(void)
...
 
 	ret = wifi_manager_get_info(&info);
 
-	TC_ASSERT_EQ("wifi_manager_get_mode_p", ret, WIFI_MANAGER_SUCCESS);
-	TC_ASSERT_EQ("wifi_manager_get_mode_p", info.mode, SOFTAP_MODE);
+	TC_ASSERT_EQ("wifi_manager_get_mode", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: info

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 5]
```diff
@@ -212,19 +212,19 @@ static void utc_wifi_manager_get_mode_p(void)
...
+	TC_ASSERT_EQ("wifi_manager_get_mode", info.mode, SOFTAP_MODE);
 	printf("\nMAC Address: ");
 	for (i = 0; i < 5; i++) {
 		printf("%x:", info.mac_address[i]);
 	}
 	printf("%x\n", info.mac_address[5]);
 	ret_cmp = strncmp(info.ssid, ap_config.ssid, strlen(CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SOFTAP_SSID));
-	TC_ASSERT_EQ("wifi_manager_get_mode_p", ret_cmp, 0);
+	TC_ASSERT_EQ("wifi_manager_get_mode", ret_cmp, 0);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: strncmp

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 6]
```diff
@@ -259,7 +259,7 @@ static void utc_wifi_manager_connect_ap_p(void)
 
 	/* change to station mode */
 	ret = wifi_manager_set_mode(STA_MODE, NULL);
-	TC_ASSERT_EQ("wifi_manager_connect_ap_p", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("wifi_manager_connect_ap", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: set_mode

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 7]
```diff
@@ -330,11 +330,11 @@ static void utc_wifi_manager_scan_ap_n(void)
 	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
 
 	ret = wifi_manager_init(&wifi_null_callbacks);
-	TC_ASSERT_EQ("wifi_manager_scan_ap_n", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("wifi_manager_scan_ap", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: init

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 8]
```diff
@@ -343,13 +343,13 @@ static void utc_wifi_manager_scan_ap_p(void)
 	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
 
 	ret = wifi_manager_deinit();
-	TC_ASSERT_EQ("wifi_manager_scan_ap_p", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("wifi_manager_scan_ap", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: deinit

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 9]
```diff
@@ -343,13 +343,13 @@ static void utc_wifi_manager_scan_ap_p(void)
...
 
 	ret = wifi_manager_init(&wifi_callbacks);
-	TC_ASSERT_EQ("wifi_manager_scan_ap_p", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("wifi_manager_scan_ap", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: init

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c [hunk 10]
```diff
@@ -46,7 +46,7 @@ static int *addr2;
 static void test_handler(int sig, tm_msg_t *info)
 {
 	int ret = strncmp(info->si_value.sival_ptr, TM_SAMPLE_MSG, strlen(TM_SAMPLE_MSG));
-	TC_ASSERT_EQ("task_manager_unicast_p", ret, 0);
+	TC_ASSERT_EQ("task_manager_unicast", ret, 0);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c]: strncmp

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c [hunk 11]
```diff
@@ -307,11 +307,11 @@ static void utc_task_manager_getinfo_with_group_p(void)
 {
 	if (!sample_info) {
 		sample_info = (task_info_t *)task_manager_getinfo_with_handle(tm_sample_handle, TM_RESPONSE_WAIT_INF);
-		TC_ASSERT_NEQ("utc_task_manager_getinfo_with_group_p", sample_info, NULL);
+		TC_ASSERT_NEQ("utc_task_manager_getinfo_with_group", sample_info, NULL);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c]: handle

**Comment by seinfra** [comment]
Target : [ea85ce25b0a6437ffbc3c54d657009bcbba497b9] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ea85ce25b0a6437ffbc3c54d657009bcbba497b9] - Code Rule Check OK.

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 12]
```diff
@@ -359,7 +359,7 @@ static void utc_wifi_manager_save_config_n(void)
 	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
 	ret = wifi_manager_save_config(NULL);
 
-	TC_ASSERT_EQ("utc_wifi_manager_save_config_n", ret, WIFI_MANAGER_INVALID_ARGS);
+	TC_ASSERT_EQ("utc_wifi_manager_save_config", ret, WIFI_MANAGER_INVALID_ARGS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: remove ```utc_```

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 13]
```diff
@@ -378,7 +378,7 @@ static void utc_wifi_manager_save_config_p(void)
 
 	ret = wifi_manager_save_config(&config);
 
-	TC_ASSERT_EQ("utc_wifi_manager_save_config_p", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("utc_wifi_manager_save_config", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: remove ```utc_```

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 14]
```diff
@@ -399,7 +399,7 @@ static void utc_wifi_manager_get_config_p(void)
 			ret = WIFI_MANAGER_FAIL;
 		}
 	}
-	TC_ASSERT_EQ("utc_wifi_manager_get_config_p", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("utc_wifi_manager_get_config", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: remove ```utc_```

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 15]
```diff
@@ -420,23 +420,23 @@ static void utc_wifi_manager_get_config_n(void)
 			ret = WIFI_MANAGER_FAIL;
 		}
 	}
-	TC_ASSERT_EQ("utc_wifi_manager_get_config_n", ret, WIFI_MANAGER_FAIL);
+	TC_ASSERT_EQ("utc_wifi_manager_get_config", ret, WIFI_MANAGER_FAIL);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: remove ```utc_```

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 16]
```diff
@@ -420,23 +420,23 @@ static void utc_wifi_manager_get_config_n(void)
...
 	TC_SUCCESS_RESULT();
 }
 
 static void utc_wifi_manager_remove_config_n(void)
 {
 	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
 	ret = wifi_manager_remove_config();
-	TC_ASSERT_EQ("utc_wifi_manager_remove_config_n", ret, WIFI_MANAGER_FAIL);
+	TC_ASSERT_EQ("utc_wifi_manager_remove_config", ret, WIFI_MANAGER_FAIL);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: remove ```utc_```

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c [hunk 17]
```diff
@@ -420,23 +420,23 @@ static void utc_wifi_manager_get_config_n(void)
...
 	TC_SUCCESS_RESULT();
 }
 
 static void utc_wifi_manager_remove_config_p(void)
 {
 	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
 	ret = wifi_manager_remove_config();
-	TC_ASSERT_EQ("utc_wifi_manager_remove_config_p", ret, WIFI_MANAGER_SUCCESS);
+	TC_ASSERT_EQ("utc_wifi_manager_remove_config", ret, WIFI_MANAGER_SUCCESS);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: remove ```utc_```

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c (hunk 11 above)
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c]: remove ```utc_```

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c (hunk 10 above)
Comment (jeongchanKim) [reply apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c]: if assert here, it is not from strncmp, but task_manager_unicast.
So I will not fix this.

File: apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c (hunk 5 above)
Comment (jeongchanKim) [reply apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c]: same as above

File: apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c [hunk 18]
```diff
@@ -287,18 +287,18 @@ static void utc_task_manager_getinfo_with_group_n(void)
 {
 	task_info_list_t *ret;
 	ret = (task_info_list_t *)task_manager_getinfo_with_group(TM_INVALID_HANDLE, TM_NO_RESPONSE);
-	TC_ASSERT_EQ("task_manager_getinfo_with_group_n", ret, NULL);
+	TC_ASSERT_EQ("task_manager_getinfo_with_group", ret, NULL);
 
 	if (!sample_info) {
 		sample_info = (task_info_t *)task_manager_getinfo_with_handle(tm_sample_handle, TM_RESPONSE_WAIT_INF);
-		TC_ASSERT_NEQ("utc_task_manager_getinfo_with_group_n", sample_info, NULL);
+		TC_ASSERT_NEQ("utc_task_manager_getinfo_with_group", sample_info, NULL);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c]: 1 . remove utc_
2. group -> handle

**Comment by seinfra** [comment]
Target : [a1410ed01d5c6f3f4500e8f6ec686e451dad75af] -

**Comment by seinfra** [comment]
Target : [a1410ed01d5c6f3f4500e8f6ec686e451dad75af] -

**Comment by seinfra** [comment]
Target : [2b02f8e476bb84b148a767497cbda82ff3107b40] -

**Comment by seinfra** [comment]
Target : [2b02f8e476bb84b148a767497cbda82ff3107b40] -

**Comment by seinfra** [comment]
Target : [df09a0fc83b707e8655f0165c07effe21c65f137] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [df09a0fc83b707e8655f0165c07effe21c65f137] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [abe034df7b5aec52b0732350681bf36b663aa86a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [abe034df7b5aec52b0732350681bf36b663aa86a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5ff914c163f45c9b95d8f99fdf7071c16eb81191] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5ff914c163f45c9b95d8f99fdf7071c16eb81191] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [023100551429bdabe6df14a4b7cbc1df5b429f90] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [023100551429bdabe6df14a4b7cbc1df5b429f90] - Code Rule Check OK.

**Pull request merged**